docs/Projects/LLM-benchmarks/project_description.md

```diff
@@ -0,0 +1,26 @@
+# LLL-Benchmark project
+
+Large Languages Models (LLM's) are becoming increasingly popular in assisting people in
+a variety of tasks. These tasks include, but are not limited to, information retrieval, assisting with
+coding and essay writing. In the context of the government, tasks can include for example supporting FOIA
+requests and aiding in answering questions of citizens.
+
+While the potential benefit of using LLM's is large, there are also significant risks.
+Basically an LLM is just a next token predictor, which bases its predictions on the user input (context) and
```

Comment:
```suggestion
Basically a LLM is just a next token predictor, which bases its predictions on the user input (context) and
```

Reply:
Wait, I think it should be "an LLM" since LLM is pronounced as "el-el-em".